### PR TITLE
Relaunch test during next week-end

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Just to see if the initial assumption of why paid containers impression were lacking is true.

### CODE REVIEW

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/paid-content-vs-outbrain.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ define([
 ], function (config) {
     return function () {
         this.id = 'PaidContentVsOutbrain';
-        this.start = '2017-02-22';
-        this.expiry = '2017-03-22';
+        this.start = '2017-02-25';
+        this.expiry = '2017-03-27   ';
```
Comment (katebee) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/paid-content-vs-outbrain.js]: Nit-pick: Spaces, although it seems the eval on the expiry (when it converts it through `Date()`) is stripping these (else I'm pretty sure a test would have failed!)

**Review by katebee — APPROVED** [review]
👍

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/paid-content-vs-outbrain.js (hunk 1 above)
Comment (regiskuckaertz) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/paid-content-vs-outbrain.js]: ha! very good, I didn't even notice. Thanks :-)

**Comment by katebee** [comment]
🚢 🚢 👍

**Pull request merged**